src/clients/QBittorrent.ts

```diff
@@ -248,6 +257,10 @@ export default class QBittorrent implements TorrentClient {
 	}
 
 	async isSubfolderContentLayout(searchee: Searchee): Promise<boolean> {
+		const { dataDirs } = getRuntimeConfig();
+		if (dataDirs.length > 0) {
+			return statSync(searchee.path).isDirectory();
+		}
```

Comment:
subfolder content layout only really takes effect for single-file torrents, but I'm inclined to say that it'll be easiest to just ignore content layouts entirely in v1 and only support `Original`. That would mean that we can just return false in this situation.